coordinator/ethereum/gas-pricing/static-cap/src/test/kotlin/net/consensys/linea/ethereum/gaspricing/staticcap/ExtraDataV1PricerServiceTest.kt

```diff
@@ -91,11 +91,17 @@ class ExtraDataV1PricerServiceTest {
       expectedVariableFees.toKWei().toUInt(),
       expectedEthGasPrice.toKWei().toUInt()
     )
-    monitor.start().thenApply {
-      vertx.setTimer(pollingInterval.inWholeMilliseconds * 2) {
+
+    // Start the service
+    monitor.start().thenAccept { _ ->
+      // Wait for a reasonable amount of time to ensure polling executes at least once
+      // Use 3x the polling interval to ensure enough time for the action to complete
+      vertx.setTimer(pollingInterval.inWholeMilliseconds * 3) {
```

Comment:
I'd even increase it as `pollingInterval.inWholeMilliseconds * 5` given that we keep `pollingInterval = 10.milliseconds`